### PR TITLE
"Buff" Vision Traits

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This takes Ultravision, Dogvision, and Normalvision, making all 3 traits into our first set of "Neutral" traits. They now cost 0 points, but still occupy one of the 5 trait selections. In the future we should have a decent roster of Positive, Neutral, and Negative traits. If Death ever finishes making Subcategories, we'll probably want to have Positive, Neutral, and Negative be the 3 "Main" categories, and all subcategories sit underneath them.

<details><summary><h1>Media</h1></summary>
<p>

Token screenshot for a 3 line PR because my maintainers will get onto me if I don't include some kind of media. :trollface: 

image

</p>
</details>

# Changelog

:cl:
- tweak: Ultraviolet Vision, Deuteranopia, and Trichromat Modification are all now 0 point Neutral traits. They still occupy one of your trait selections.

### CODE REVIEW

**Review by DEATHB4DEFEAT — APPROVED** [review]

File: Resources/Prototypes/DeltaV/Traits/altvision.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 - type: trait
   id: UltraVision
   category: Visual
-  points: -1
+  points: 0
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Traits/altvision.yml:4]:
```suggestion
```

File: Resources/Prototypes/DeltaV/Traits/altvision.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@
 - type: trait
   id: DogVision
   category: Visual
-  points: -1
+  points: 0
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/DeltaV/Traits/altvision.yml:21]:
```suggestion
```

File: Resources/Prototypes/Traits/neutral.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@
 - type: trait
   id: NormalVision
   category: Visual
-  points: -1
+  points: 0
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Traits/neutral.yml:34]:
```suggestion
```

**Review by DEATHB4DEFEAT — COMMENTED** [review]
Forgot to remember to tell you you can do this

**Review by OldDanceJacket — APPROVED** [review]

**Pull request merged**